src/Collection/Fulfilment/OrderNotesCollection.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelNL\Sdk\src\Collection\Fulfilment;
+
+use MyParcelNL\Sdk\src\Concerns\HasApiKey;
+use MyParcelNL\Sdk\src\Concerns\HasUserAgent;
+use MyParcelNL\Sdk\src\Model\Fulfilment\OrderNote;
+use MyParcelNL\Sdk\src\Model\MyParcelRequest;
+use MyParcelNL\Sdk\src\Model\RequestBody;
+use MyParcelNL\Sdk\src\Support\Arr;
+use MyParcelNL\Sdk\src\Support\Collection;
+
+class OrderNotesCollection extends Collection
+{
+    use HasUserAgent;
+    use HasApiKey;
+
+    /**
+     * @return array Indexed array of order uuids present in this collection.
+     */
+    private function orderUuidsInCollection(): array
+    {
+        return array_unique($this->reduce(
```

Comment:
kan dit niet?

```php
return $this
    ->map(function (OrderNote $orderNote) {
        return $orderNote->getOrderUuid();
    })
    ->unique();
```

en dan return je meteen de hele collection ipv er een array, dat kan ook pas op het moment dat je hem gebruikt

(Dit kan niet in de pdk omdat `map` een `new static()` returned, wat problematisch wordt door de collection casts die we daar hebben. Dat moet ook nog een keer aangepakt worden, maar dan moet er een andere oplossing op bedacht worden 🤔)